images/sunshine/Dockerfile

```diff
@@ -4,7 +4,6 @@ ARG BASE_APP_IMAGE
 FROM ubuntu:22.04 AS sunshine-builder
 
 ARG DEBIAN_FRONTEND=noninteractive
-ARG BASE_APP_IMAGE
```

Comment:
i think this is required to make the `BASE_APP_IMAGE` arg available in the second stage, but i could be wrong

Reply:
just noticed you addressed this in the commit message. if it isn't needed, that's awesome!